flutter/lib/src/view_hierarchy/sentry_tree_walker.dart

```diff
@@ -54,16 +54,17 @@ class _TreeWalker {
 
     final renderObject = element.renderObject;
     if (renderObject is RenderBox) {
-      final offset = renderObject.localToGlobal(Offset.zero);
-      if (offset.dx > 0) {
-        x = offset.dx;
-      }
-      if (offset.dy > 0) {
-        y = offset.dy;
-      }
+      final offset = renderObject.globalToLocal(Offset.zero);
```

Comment:
@ueman ideas why `globalToLocal` return negative values as well?
Found https://stackoverflow.com/questions/36800042/globaltolocal-not-behaving-as-i-expected-pretty-sure-im-just-not-getting-it but the Flutter docs don't state that.
Wondering if I should use something else instead of `Offset.zero`.
We need to get the `x` and `y` relative to the parent.